xbmc/guilib/GUILabel.h

```diff
@@ -245,4 +245,5 @@ class CGUILabel
   CRect          m_maxRect;      ///< maximum sizing of text
   bool           m_invalid;      ///< if true, the label needs recomputing
   COLOR          m_color;        ///< color to render text \sa SetColor, GetColor
+  unsigned int   m_maxScrollLoops = ~0;
```

Comment:
Can this be `int m_maxScrollLoops = -1;` to use a default that doesn't rely on variable size?